From 7d7f8f8e2c2c5b2321af8422adb858fee4d3798e Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Thu, 21 Jun 2018 12:47:56 -0700
Subject: [PATCH] TRITON-530 nightly-1 vmapi fabric provision test is failing
 on "should have found one NAT vm" Reviewed by: Dave Eddy
 <dave.eddy@joyent.com> Approved by: Dave Eddy <dave.eddy@joyent.com>

---
 test/vms.full.test.js | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/vms.full.test.js b/test/vms.full.test.js
index 01e11b3..243fb6e 100644
--- a/test/vms.full.test.js
+++ b/test/vms.full.test.js
@@ -2798,8 +2798,10 @@ exports.destroy_test_vms_final = function (t) {
 exports.find_fabric_network = function (t) {
     assert.arrayOfObject(NETWORKS, 'NETWORKS');
 
+    // Find a fabric network that is owned by CUSTOMER.
     fabricNetwork = NETWORKS.find(function _findFabricNetwork(n) {
-        return n.fabric === true;
+        return n.fabric === true && Array.isArray(n.owner_uuids) &&
+            n.owner_uuids.indexOf(CUSTOMER) >= 0;
     });
 
     t.ok(fabricNetwork, 'found a fabric network');
-- 
2.21.0

